src/main/java/com/kkato/helloworld/HelloworldController.java

```diff
@@ -2,20 +2,18 @@
 
 import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 
 public class HelloworldController {
     @GetMapping("/Hello world")
-    public String Hello() {
-        return "Hello world";
+    public HelloworldResponse Hello() {
+        return new HelloworldResponse("Hello world");
     }
-
-    @DeleteMapping("/Hello world2")
-    public String Hello2(){
-        return "Hello kato";
+    @PatchMapping("/Say Hello project")
```

Comment:
スペースはプログラミングでは区切り文字として扱うこともおおいのでパスにいれないほうがいいですね。
```suggestion
    @PatchMapping("/say-hello-project")
```
のようにケバブケースにするのがいいでしょう。
たまにですがスネークケースのAPIもありますね。

Reply:
ご指摘ありがとうございます！
下記の通り修正いたしましたのでご確認をお願いいたします。
↓↓
1793a43

①PatchメソッドでURLのパス（/say-hello-project）で動作することを確認。【HTTPステータス:200】
  合わせて、以前のパス（/Say hello project）でエラーが出ることを確認。【HTTPステータス:404】
![スクリーンショット (42)](https://github.com/Kazuyuki-Kato/six-assignment/assets/154575590/cf17708c-06c2-4539-8c3a-e2cd899c02bb)
![スクリーンショット (43)](https://github.com/Kazuyuki-Kato/six-assignment/assets/154575590/119392dc-5a90-45c4-8461-552738c58587)

②GETメソッドでURLのパス（/hello-world）で動作することを確認。【HTTPステータスコード:200】
 合わせて、以前のパス（/Hello world）でエラーが出ることを確認。【HTTPステータスコード:404】
![スクリーンショット (44)](https://github.com/Kazuyuki-Kato/six-assignment/assets/154575590/98eedf7e-240e-4a58-b7bb-b5228a04d52f)
![スクリーンショット (45)](https://github.com/Kazuyuki-Kato/six-assignment/assets/154575590/af9d095e-edd8-48ab-b063-7dd72c1b5fc3)

ご確認の程宜しく御願い致します！

Reply:
OKです！